components/GameScreen.vue

```diff
@@ -56,6 +64,83 @@ type DrawBotArgs = {
   graphics: Graphics;
 };
 
+function smoothZoom(Pos: { x: number; y: number }, scale: number) {
```

Comment:
nit: variables should be lowercase

```suggestion
function smoothZoom(pos: { x: number; y: number }, scale: number) {
```

Reply:
Also, let's prefer named variables because they are generally make the calling code a bit more readable.

```suggestion
function smoothZoom({ pos, scale }: { pos: { x: number; y: number }, scale: number }) {
```